web-client/src/modules/settings/components/SettingsList/SettingsList.tsx

```diff
@@ -69,8 +70,8 @@ const SettingsList: React.FC<SettingsProps> = ({
             </SettingsListCollapsePanel>
           </Collapse>
         </Col>
-      </Row>
-      <Row gutter={[0, 12]}>
+      </SettingsListItemWrapper>
+      <SettingsListItemWrapper topSpace="40px">
```

Comment:
Could you have this stack up one on top of each other using padding and flexbox rather than with top css?
This isn't a very scalable solution for the UI

Reply:
resolved this in latest commit.

Reply:
Hey @Tapudp , great job on fixing the main CSS issue. It works flawlessly on different screens. 
However, this last one about stacking up one on top of another is still not done.